tests/setup-softhsm.sh

```diff
@@ -282,6 +305,10 @@ export ECPEERBASEURI="${ECPEERBASEURI}"
 export ECPEERPUBURI="${ECPEERPUBURI}"
 export ECPEERPRIURI="${ECPEERPRIURI}"
 export ECPEERCRTURI="${ECPEERCRTURI}"
+export EDBASEURI="${EDBASEURI}"
```

Comment:
missing `export EDBASEURIWITHPIN="${EDBASEURIWITHPIN}"` on the line above

Reply:
none of the EC stuff seem to export it
and we do not use it, so I will just drop it for now